### PR TITLE
[ja] translate components/cookie.rst

### PR DESCRIPTION
**Pull request by iKenji**
Sorry,  I closed p-r by mistake.
This is the same #4199.

@okinaka I fixed. Thank you for pointing out.

### CODE REVIEW

File: ja/controllers/components/cookie.rst [hunk 1]
```diff
@@ -1,20 +1,121 @@
...
+path
+    Cookie を有効にするパスを設定します。もしパスが '/foo/' で設定されていた場合、
+    Cookie は /foo/ ディレクトリとサブディレクトリ(例: /foo/bar/ )でのみ有効になります。
+    デフォルトは、アプリケーションの基本パスです。
+domain
+    Cookie が有効なドメイン。
+    example.com の全てのサブドメインで Cookie を有効にするには、 '.example.com'に
+    ドメインをセットしてください。
+secure
+    Cookie はセキュアな HTTPS 接続を通してのみ指定すべきです。このオプションを
```
Comment (chinpei215) [inline ja/controllers/components/cookie.rst]: > Indicates that the cookie should only be transmitted over a secure HTTPS connection.

The grammatical structure of the above sentence is:
**S** (secure)
**V** Indicates
**O** that the cookie should only be transmitted over a secure HTTPS connection.

Cookie がセキュアな HTTPS 上でのみ転送されるべきであること (=O) を示します (=V)。

File: ja/controllers/components/cookie.rst [hunk 2]
```diff
@@ -1,20 +1,121 @@
...
+    ``true`` に設定した場合、 Cookie はセキュアな接続時のみ発行されるようになります。
+key
+    暗号化された Cookie が有効なときは、暗号化キーが使われます。
```
Comment (chinpei215) [inline ja/controllers/components/cookie.rst]: > Encryption key used when encrypted cookies are enabled.

Probably, the above sentence doesn't have S and V.

暗号化された Cookie が有効な時に使われる暗号化キー。

File: ja/controllers/components/cookie.rst [hunk 3]
```diff
@@ -1,20 +1,121 @@
...
+Cookie コンポーネントは、いくつかのメソッドを提供します。
+
+.. php:method:: write(mixed $key, mixed $value = null)
+
+    write() は Cookie コンポーネントの核となるメソッドです。 $key は Cookie の
+    変数名で、$value は保存されるデータです。::
+
+        $this->Cookie->write('name', 'Larry');
+
+    変数名をドット記法することで、変数をグループで扱うこともできます。::
```
Comment (chinpei215) [inline ja/controllers/components/cookie.rst]: > by using dot notation in the  key parameter

キーのパラメータにドット記法を使うことで、

File: ja/controllers/components/cookie.rst [hunk 4]
```diff
@@ -1,20 +1,121 @@
...
+
+        // ドット記法でグループ化された変数は、次のように
+        // 配列として取得されます。
+        $this->Cookie->read('User');
+
+        // ['name' => 'Larry', 'role' => 'Lead']
+
+.. php:method:: check($key)
+
+    :param string $key: The key to check.
```
Comment (chinpei215) [inline ja/controllers/components/cookie.rst]: チェックするキー

**Comment by iKenji** [comment]
@chinpei215 
I fixed it.
Thank you for pointing out and advice.
I'll study English harder.

**Comment by chinpei215** [comment]
Looks good to me. Nice work!

**Pull request merged**